lib/erlef/data/schema/academic_paper.ex

```diff
@@ -0,0 +1,60 @@
+defmodule Erlef.Data.Schema.AcademicPaper do
+  @moduledoc """
+  Schema module for academic papers
+  """
+  use Erlef.Data.Schema
+
+  @type t :: %__MODULE__{
+          id: binary(),
+          title: String.t(),
+          author: String.t(),
+          language: String.t(),
+          url: String.t(),
+          pay_wall?: :boolean | nil,
+          description: String.t() | nil,
+          technologies: {:array, String.t()} | nil,
+          original_publish_date: Date.t() | nil,
+          published_at: :utc_datetime | nil,
+          deleted_at: :utc_datetime | nil
+        }
+
+  @required_fields ~w|
+    title
+    author
+    language
+    url
+  |a
+
+  @optional_fields ~w|
+    pay_wall?
+    description
+    technologies
+    original_publish_date
+    published_at
+    deleted_at
+  |a
+
+  @all_fields @required_fields ++ @optional_fields
+
+  schema "academic_papers" do
+    field(:title, :string)
+    field(:author, :string)
+    field(:language, :string)
+    field(:url, :string)
+    field(:pay_wall?, :boolean)
+    field(:description, :string)
+    field(:technologies, {:array, :string})
+    field(:original_publish_date, :date)
+
```

Comment:
Do we need an submitted_by and approved_by? I think we will if members are to be able to submit papers.

Reply:
Debated on adding those but figured we could add them when the ability to add through the app is in place

Reply:
Sure.